sdk/storage/src/table/query_result.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{ContinuationToken, TableEntity};
+use azure_core::errors::AzureError;
+use http::HeaderMap;
+use hyper::body;
+use serde::{de::DeserializeOwned, Serialize};
+
+pub struct QueryResult<T>
+where
+    T: DeserializeOwned,
+{
+    pub entities: Vec<TableEntity<T>>,
+    pub continuation_token: Option<ContinuationToken>,
+}
+
+impl<T> std::convert::TryFrom<(url::Url, &HeaderMap, &body::Bytes)> for QueryResult<T>
```

Comment:
I always find conversion methods from tuples to be very confusing unless it's an extremely obvious one like `(f32, f32)` to `Point`. Is there a better way to model this?